helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```
@@ -91,6 +91,22 @@ spec:
         cancel_if_outside_working_hours: "true"
         severity: page
         topic: releng
+    - alert: ClusterCapiAppFailed
+      annotations:
+        description: '{{`Cluster CAPI App {{ if $labels.exported_namespace }}{{ $labels.exported_namespace }}{{ else }}{{ $labels.namespace }}{{ end }}/{{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
```

```suggestion
        description: '{{`Cluster CAPI App {{ if $labels.exported_namespace }}{{ $labels.exported_namespace }}{{ else }}{{ $labels.namespace }}{{ end }}/{{ $labels.name }}, version {{ $labels.version }} is in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
```

according the test-data the `{{if $label.status}}` seems wrong/obsolete?

---

i did run the `make test` and it succeded. commiting your suggestion :+1:

---

yes, the test didn't fail because your test-case was "wrong" (you have two spaces between `is` and `in`)

---

ah ! :D